examples/webgl_furnace_test.html

```diff
@@ -48,32 +48,33 @@
 
 					scene.traverse( function ( child ) {
 
-						if ( child.isMesh ) child.material.color.setHex( 0xccccff ); // tinted for visibility
-						render();
+						if ( child.isMesh ) child.material.color.setHex( 0xffffff );
 
 					} );
 
+					render();
+
 				} );
 
 				document.body.addEventListener( 'mouseout', function () {
 
 					scene.traverse( function ( child ) {
 
-						if ( child.isMesh ) child.material.color.setHex( 0xffffff );
-						render();
+						if ( child.isMesh ) child.material.color.setHex( 0xccccff ); // tinted for visibility
 
 					} );
 
+					render();
+
 				} );
 
 				// scene
 
 				scene = new THREE.Scene();
 
 				// camera
-
-				camera = new THREE.OrthographicCamera( frustumSize * aspect / - 2, frustumSize * aspect / 2, frustumSize / 2, frustumSize / - 2, 1, 30 );
-				camera.position.set( 0, 0, 10 );
+				camera = new THREE.PerspectiveCamera( 40, aspect, 1, 30 );
```

Comment:
The code in `onWindowResize()` still thinks an orthographic camera is used.

Reply:
fixed!